### ISSUE TITLE
Classification sample has no learning_rate and n_iter_no_change

### ISSUE THREAD

**Issue by GehrkeSven** [MathiasKraus/igann#19]
In the classifier example given, there are no parameters for learning rate and n_iter_no_change, therefor the example isnt working as is.

igann_classifier.get_params()

{'task': 'classification', 'n_hid': 10, 'elm_scale': 1, 'elm_alpha': 1, 'init_reg': 1, 'act': 'elu', 'n_estimators': 5000, 'early_stopping': 50, 'sparse': 0, 'device': 'cpu', 'random_state': 1, 'optimize_threshold': False, 'verbose': 0, 'boost_rate': 0.1}

**Comment by MathiasKraus** [comment]
Thank you for reporting this bug! In an earlier version of igann, the parameters were named `learning_rate` and `n_iter_no_change`. In the latest version, we've updated these names to `boost_rate` and `early_stopping` to align with the conventions used in other packages. We also updated the readme accordingly.
